requirements_generate_data.txt

```diff
@@ -1,3 +1,3 @@
 -r requirements_base.txt
-awscli
-homeassistant
+awscli==1.32.80
+homeassistant==2024.3.3
```

Comment:
This is not a mistake, as the action is currently using Python 3.11 this is the newest version that can be used.